### DISCUSSION TITLE
Urgent: My GitHub Account Has Been Hacked, 2FA violated

### DISCUSSION THREAD

**Discussion by egosinenominebak** [community/community#126689] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub Community,

I am writing to inform you about a serious issue regarding my GitHub account being compromised.

Context:
I recently received an email that an unfamiliar email address ([brant61hageneszev@hotmail.com](mailto:brant61hageneszev@hotmail.com)) was added to my account. Here is the email I received for reference:

_________________________________________________________________________

**Email from GitHub:**

Hey egosinenomine!

An email address ([brant61hageneszev@hotmail.com](mailto:brant61hageneszev@hotmail.com)) was added to your account. Visit [GitHub Email Settings](https://github.com/settings/emails) to review email addresses currently associated with your account.

To see this and other security events for your account, visit [Security Log](https://github.com/settings/security-log).

If you run into problems, please contact support by visiting [GitHub Contact Support](https://github.com/contact).

Thanks,
The GitHub Team

_______________________________________________________________________________________

Actions Taken:

Reviewed Email Addresses: I checked the email addresses associated with my account and noticed the unfamiliar email.
Security Log: I reviewed the security logs for any other suspicious activities.
Contacted GitHub Support: I have reached out to GitHub Support through their contact page [GitHub Contact Support](https://github.com/contact). No response yet.

**Additional Issues:**

The unauthorized individual not only added a new email address but also changed the password and removed both my personal and institutional email accounts associated with my GitHub account.

Request for Assistance:

I am seeking immediate assistance to regain control of my account and ensure the security of my repositories and data. If anyone from the community or the GitHub team can provide additional guidance or expedite the support process, it would be greatly appreciated.

Thank you for your understanding and support.

egosinenomine.

**Comment by ebndev** [comment]
Hi @egosinenominebak,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
